src/script/event/WebSocketService.js

```diff
@@ -41,7 +41,7 @@ z.event.WebSocketService = class WebSocketService {
 
   static get CONFIG() {
     return {
-      PING_INTERVAL: z.util.TimeUtil.UNITS_IN_MILLIS.SECOND * 30,
+      PING_INTERVAL: z.util.TimeUtil.UNITS_IN_MILLIS.SECOND * 5,
       PING_INTERVAL_THRESHOLD: z.util.TimeUtil.UNITS_IN_MILLIS.SECOND * 2,
```

Comment:
Please note, I increased the amount of 'ping' requests, because we have to wait for the `PING_INTERVAL_THRESHOLD` before Wire detects network loss, and it was a whole minute. Now it is only 10 seconds, which I personally find acceptable.

The amount of traffic is not increased as much, because we also changed from sending 36 bytes per message (`Wire is so much nicer with internet!`) to only 4 (`ping`).